src/Services/CategoriesService.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+
+const CATEGORIES_URL = 'http://ongapi.alkemy.org/api/categories';
+
+const getAllCategories = async () => {
```

Comment:
instead of have a getAll and a get by id, use only a get and if the id is defined then add it to the url, else use only the url